### PR TITLE
Options Constants: Accessibility Cleanup

### PR DESCRIPTION
**Pull request by Windchild292**
These have been an accessibility problem for a while, this fixes those issues

### CODE REVIEW

**Comment by pheonixstorm** [comment]
Looks good, will likely cause conflicts with Cross Country and Environmental Specialist, but those can be easily fixed.

File: megamek/src/megamek/common/options/OptionsConstants.java [hunk 1]
```diff
@@ -21,112 +21,111 @@ public class OptionsConstants {
...
-    public static final String QUIRK_NEG_UNBALANCED= "unbalanced"; 
-    public static final String QUIRK_NEG_UNSTREAMLINED= "unstreamlined"; 
+    public static final String QUIRK_NEG_POOR_WORK = "poor_work"; 
+    public static final String QUIRK_NEG_PROTOTYPE = "prototype"; 
+    public static final String QUIRK_NEG_RAMSHACKLE = "ramshackle"; 
+    public static final String QUIRK_NEG_SENSOR_GHOSTS = "sensor_ghosts"; 
+    public static final String QUIRK_NEG_SUSCEPTIBLE_CWS = "susceptible_cws"; 
+    public static final String QUIRK_NEG_UNBALANCED = "unbalanced"; 
+    public static final String QUIRK_NEG_UNSTREAMLINED = "unstreamlined"; 
     public static final String QUIRK_NEG_WEAK_HEAD_1= "weak_head_1";
```
Comment (SJuliez) [inline megamek/src/megamek/common/options/OptionsConstants.java:121]: Forgot some :D

File: megamek/src/megamek/common/options/OptionsConstants.java [hunk 2]
```diff
@@ -272,260 +262,251 @@ public class OptionsConstants {
...
+    public static final String INIT_PROTOS_MOVE_MULTI = "protos_move_multi";
+    public static final String INIT_INF_PROTO_MOVE_MULTI = "inf_proto_move_multi";
+    public static final String INIT_SIMULTANEOUS_DEPLOYMENT = "simultaneous_deployment";  
+    public static final String INIT_SIMULTANEOUS_MOVEMENT = "simultaneous_movement";  
+    public static final String INIT_SIMULTANEOUS_TARGETING = "simultaneous_targeting";  
+    public static final String INIT_SIMULTANEOUS_FIRING = "simultaneous_firing";  
+    public static final String INIT_SIMULTANEOUS_PHYSICAL = "simultaneous_physical";  
+    public static final String INIT_FRONT_LOAD_INITIATIVE = "front_load_initiative";  
+    public static final String INIT_INITIATIVE_STREAK_COMPENSATION = "initiative_streak_compensation";
+    public static final String RPG_PILOT_ADVANTAGES = "pilot_advantages";
```
Comment (SJuliez) [inline megamek/src/megamek/common/options/OptionsConstants.java:502]: Is removing those blank lines between groups reaaaaally an accessibility improvement?

**Review by SJuliez — APPROVED** [review]

File: megamek/src/megamek/common/options/OptionsConstants.java (hunk 1 above)
Comment (Windchild292) [reply megamek/src/megamek/common/options/OptionsConstants.java:121]: I did not expect game options with names ending with numbers, so I missed them. Fixed.

File: megamek/src/megamek/common/options/OptionsConstants.java (hunk 2 above)
Comment (Windchild292) [reply megamek/src/megamek/common/options/OptionsConstants.java:502]: Those aren't different groups, they're all game options.

**Pull request merged**